Sources/Collections/SkeletonCollectionDelegate.swift

```diff
@@ -41,6 +41,11 @@ extension SkeletonCollectionDelegate: UITableViewDelegate {
 
         return nil
     }
+
+    func tableView(_ tableView: UITableView, didEndDisplaying cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        cell.hideSkeleton()
```

Comment:
Why we need to hide the skeleton? please, give me some more info :)

Reply:
If table view's frame changes after `showSkeleton` call, then `visibleCells` may return fewer cells in `hideSkeleton` call. Then some of cells will not receive `hideSkeleton` and will be reused in skeleton state.

Reply:
Ok, now I understand, thanks :)